src/features/landing/components/Possibilities/constants.ts

```
@@ -1,61 +1,10 @@
-export const imageList: ImageList[] = [
-  { image: Wix, alt: 'Wix' },
-  { image: WordPress, alt: 'WordPress' },
-  { image: Zapier, alt: 'Zapier' },
-  { image: Shopify, alt: 'Shopify' },
-  { image: Magento, alt: 'Magento' },
-  { image: Joomla, alt: 'Joomla' },
-  { image: Drupal, alt: 'Drupal' },
-  { image: WooCommerce, alt: 'WooCommerce' },
-];
+
+export const cardList: CardItem[] = getWhyUsCards().map(card => ({
```

ESLint

[error] More than 2 blank lines not allowed.

(no-multiple-empty-lines)